homeassistant/components/mqtt/fan.py

```diff
@@ -137,6 +147,7 @@ def valid_speed_range_configuration(config):
             vol.Optional(CONF_PERCENTAGE_COMMAND_TEMPLATE): cv.template,
             vol.Optional(CONF_PERCENTAGE_STATE_TOPIC): mqtt.valid_subscribe_topic,
             vol.Optional(CONF_PERCENTAGE_VALUE_TEMPLATE): cv.template,
+            vol.Optional(CONF_PAYLOAD_HIGH_SPEED, default=SPEED_HIGH): cv.string,
```

Comment:
This seems to be unintentionally duplicated?

Reply:
Removed the duplicated line.